frontend/src/app/components/ExperimentCardList/index.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../../styles/_utils';
```

Comment:
The [.card-list-heading](https://github.com/mozilla/testpilot/blob/918ab136e99c35fd6720e4453e8825209d15eb6c/frontend/src/app/components/ExperimentCardList/index.scss#L18) could use a font tweak:
```
font-weight: 300;
font-size: 33px;
text-shadow: 0 2px 0 $transparent-black-2;
```
Also [this heading looks horizontally centered now in the spec](https://mozilla.github.io/testpilot-assets/Test_Pilot_Website/01_Measurements/Homepage_20170927/#artboard8).